server/data/search.go

```diff
@@ -33,20 +19,36 @@ func SearchUsers(field string, query string) []*UserProfile {
 				loud_tier,
 				coed
 		FROM users
-		WHERE %s=$1
-		`, searchField)
+		`
 
-	rows, err := db.Query(context.Background(), sqlQuery, &query)
+	// Add AND WHERE clauses in SQL string
+	for idx, slice := range queryFields {
+		field := slice[0]
+		value := slice[1]
 
-	if err != nil {
-		fmt.Println("Error searching for users in field:", searchField, "query:", query)
+		// build clauses dynamically
+		clause := "WHERE"
+		if idx > 0 {
+			clause = "AND"
```

Comment:
Hey, I tried doing it using "AND WHERE" but it would return me an error and was looking for documentation in Postgres. Do you have an example on how to do it this way?